manageiq-operator/pkg/controller/manageiq/manageiq_controller.go

```diff
@@ -50,8 +50,49 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	if err != nil {
 		return err
 	}
-
 	// Watch for changes to secondary resource Deployments and requeue the owner Manageiq
+	err = c.Watch(&source.Kind{Type: &corev1.PersistentVolumeClaim{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    &miqv1alpha1.Manageiq{},
+	})
+	if err != nil {
+		return err
+	}
+	err = c.Watch(&source.Kind{Type: &corev1.Secret{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    &miqv1alpha1.Manageiq{},
+	})
+	if err != nil {
+		return err
+	}
+	err = c.Watch(&source.Kind{Type: &corev1.ConfigMap{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    &miqv1alpha1.Manageiq{},
+	})
+	if err != nil {
+		return err
+	}
+	err = c.Watch(&source.Kind{Type: &corev1.Service{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    &miqv1alpha1.Manageiq{},
+	})
+	if err != nil {
+		return err
+	}
+	err = c.Watch(&source.Kind{Type: &corev1.ServiceAccount{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    &miqv1alpha1.Manageiq{},
+	})
+	if err != nil {
+		return err
+	}
+	err = c.Watch(&source.Kind{Type: &rbacv1.RoleBinding{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    &miqv1alpha1.Manageiq{},
```

Comment:
Can we deduplicate this handler reference? Maybe have one
```go
ownerHandler := &handler.EnqueueRequestForOwner{IsController: true, OwnerType:    &miqv1alpha1.Manageiq{}}
```

At the start and call `Watch` with the same one each time?